packages/Layout/header/src/NavBar/NavBar.js

```diff
@@ -39,10 +39,6 @@ class NavBar extends Component {
     };
   }
 
-  componentWillReceiveProps(nextProps) {
```

Comment:
is there any impact to delete this lifecycle method ?

Reply:
should be replace by getDerivedStateFromProps, what you think ?

Reply:
I use getDerivedStateFromProps but doesn't work (tests fail). But are you agree that if props change, component will be rerender & state with init prop will be updated ?

Reply:
you're right, i think is useless here : https://fr.reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html